src/classes/genre.rb

```diff
@@ -0,0 +1,17 @@
+require 'securerandom'
+
+class Genre
+  attr_accessor :name, :items
+  attr_reader :id
+
+  def initialize(name, id = SecureRandom.uuid)
+    @id = id
+    @name = name
+    @items = []
+  end
```

Comment:
Kindly change the `@id` initial value for `SecureRandom.uuid` instead of using a _default parameter_. Because `@id` is a private instance variable according to the UML diagram. It avoids the user to input a wrong value in this.

![image](https://user-images.githubusercontent.com/106117541/212936669-d72b2b6e-7364-4ee6-ad00-a5f5590f6940.png)